docs/docs/999-faq/040-alternatives/041-terraform-vs-wing.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: What are the main differences between Wing and Terraform?
+title: Terraform vs. Wing [Main differences]
```

Comment:
How does this render in the sidebar? Feels like it's going to be a mess.

Reply:
This page already has sidebar_label set to just show as "Terraform"